CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+| pr-changes-requested | A project maintainer has requested changes and is awaiting updates from the author      |
+| pr-expired           | Requested changes were not provided within an expected time window.                     |
+| pr-in-review         | A project maintainer is reviewing the pull request                                      |
+
+## Creating documentation and samples
+
+There are a variety of ways you can contribute to Functions content. In addition to those discussed in this section, updates to repository readmes, wikis, or code comments are always welcome.
+
+The primary Functions documentation on [docs.microsoft.com](https://docs.microsoft.com) is open source on [MicrosoftDocs/azure-docs](https://github.com/MicrosoftDocs/azure-docs). You can learn more about contributing to the docs at the repo or by following the [Microsoft Docs Contributor Guide](https://docs.microsoft.com/contribute/). 
+
+The [Serverless Community Library](https://serverlesslibrary.net) is a catalog of community samples that can be deployed right away. You can submit a sample to the library by following the [Serverless Library contribution guide](https://github.com/Azure/ServerlessLibrary#how-to-contribute-to-the-serverless-library). In addition to samples, most client experiences for creating functions make use of a function template. The [templates repository](Azure/azure-functions-templates) defines templates for each supported trigger and language, as well as core sample scenarios.
+
+## Azure Functions repositories
+
+Azure Functions is built on work from the following repositories:
+
+| Category            | Repository                                                                                                          | Description                                                                                                                    |
+|---------------------|---------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| General             | [Azure/azure-functions](https://github.com/Azure/azure-functions)                                                   | General information and issue management                                                                                       |
+| General             | [Azure/azure-functions-premium-plan](https://github.com/Azure/azure-functions-premium-plan)                         | Documentation and issue management related to the Azure Functions Premium Plan                                                 |
+| Content             | [MicrosoftDocs/azure-docs](https://github.com/MicrosoftDocs/azure-docs)                                             | Developer documentation found on [docs.microsoft.com](https://docs.microsoft.com)                                              |
```

Comment:
This is for all of Azure. Could we point them to something more specific to functions?

Reply:
There doesn't exist anything more specific repo-wise. We can modify the link to "Developer documentation found on https://docs.microsoft.com/azure/azure-functions" if you think that's better, but honestly I'm content (pun intended) with keeping it as-is. Maybe we call out that it includes more than just Functions in the description, though?

I was also toying with the idea of describing the workflow from docs.com using the edit button on a given page. That would be a more direct approach for content contributors. Thoughts?

Reply:
I understand that it's one repo. My concern is that this link isn't particularly actionable to the user since it doesn't point to anything specific to Functions.

The Edit button workflow is definitely advisable. I prefer to use that myself, honestly.